luxonis_train/nodes/efficient_bbox_head.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         n_heads: Literal[2, 3, 4] = 3,
         conf_thres: float = 0.25,
         iou_thres: float = 0.45,
+        max_det: int = 300,
```

Comment:
So I think honestly this is a unique parameter of NN archive which is more for depthai or user preference. I don't think it needs to be a head parameter. We could set it as a constant somewhere else and assume users can change it if desired.

CC @kozlov721 for your opinion as well.

Reply:
We can then say the same about `conf_thres` and `iou_thres` as those are all parameters of `nms`. The reason the `nms` parameters are head parameters is to ensure the same parameters will be used for all metrics / visualizers etc. and we don't need to specify it for each of them separately.

Reply:
Oh my bad I didn't realize they're used in visualizers - makes sense. Disregard my comment